virt/virt-2-6-release-notes.adoc

```diff
@@ -82,7 +83,10 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //New known issues for 2.6 (WIP)
 
-//CNV-9288: Blank block disks and VMware imported disk images are not preallocated even if requested
+* Some CDI operations are currently not preallocated when requested. These include:
```

Comment:
Is it possible to link to the related BZ or documentation for this issue?

Reply:
No bug to speak of (I think it was issue within epic) but I can add an xref to our preallocation docs